package.json

```diff
@@ -15,25 +15,25 @@
     "prepare": "husky"
   },
   "dependencies": {
-    "@types/history": "^5.0.0",
-    "@types/node": "^20.11.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-double-marquee": "^1.1.0",
     "react-responsive-carousel": "^3.2.23",
-    "react-router-dom": "^6.21.2"
+    "react-router-dom": "^6.22.3"
   },
   "devDependencies": {
-    "@testing-library/jest-dom": "^6.2.0",
-    "@testing-library/react": "^14.1.2",
+    "@testing-library/jest-dom": "^6.4.2",
+    "@testing-library/react": "^15.0.2",
     "@testing-library/user-event": "^14.5.2",
-    "@types/jest": "^29.5.11",
-    "@types/react": "^18.2.43",
-    "@types/react-dom": "^18.2.17",
-    "@typescript-eslint/eslint-plugin": "^6.21.0",
-    "@typescript-eslint/parser": "^6.21.0",
+    "@types/history": "^5.0.0",
```

Comment:
Pay attention the this message:
![image](https://github.com/rolling-scopes/site/assets/69504277/9724b1a8-c54e-4d7f-92b0-cad87c78e6e6)

Reply:
Fixed